spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java

```diff
@@ -65,32 +62,14 @@ void testGetLoggingOptions() {
         .isNotNull()
         .contains("spring-cloud-gcp-logging")
         .contains("Spring");
```

Comment:
Looking at the implementation of `LoggingAppender.java`, it only override `getLoggingOptions` method. I think the proper tests should verify whether the logging options are correctly set.